test/passing/tests/override2.ml

```diff
@@ -0,0 +1,34 @@
+let _ =
+  object
+    val mutable v = 0
+
+    val mutable l = []
+
+    val mutable o = None
+
+    method inc = {<v = v + 1>}
+
+    method empty = {<l = [] >}
+
+    method singleton x = {<l = [x] >}
+
+    method both x = {<l = [x]; v = x>}
+
+    method both_ws x = {<v = x; l = [x] >}
+
+    method with_let1 x =
+      {< l = let p = [x] in
+             p >}
+
+    method with_let2 x =
+      {< l = let a = x + v in
+             [a]  >}
+
+    method update_constr_none = {<o = None>}
+
+    method update_constr_some_empty = {<o = Some [] >}
+
+    method update_constr_some x = {<o = Some [x] >}
+
+    method prepend x = {<l = x :: l>}
+  end
```

Comment:
Probably also a test for `type t = <x : [`A]>`, which I think is legal syntax we need to print with a space (but probably already do?)